docs/settings.rst

```diff
@@ -291,6 +291,58 @@ Options that change ticket properties
             (7, _('7. Hot')),
         )
 
+Time Tracking Options
+---------------------
+
+- **HELPDESK_FOLLOWUP_TIME_SPENT_AUTO** If ``True``, calculate follow-up 'time_spent' with previous follow-up or ticket creation time.
+
+  **Default:** ``HELPDESK_FOLLOWUP_TIME_SPENT_AUTO = False``
+
+- **HELPDESK_FOLLOWUP_TIME_SPENT_OPENING_HOURS** If defined, calculates follow-up 'time_spent' according to open hours.
+  
+  **Default:** ``HELPDESK_FOLLOWUP_TIME_SPENT_OPENING_HOURS = {}``
+  
+  If HELPDESK_FOLLOWUP_TIME_SPENT_AUTO is ``True``, you may set open hours to remove off hours from 'time_spent'::
+  
+        HELPDESK_FOLLOWUP_TIME_SPENT_OPENING_HOURS = {
+            "monday": (8.5, 19),
+            "tuesday": (8.5, 19),
+            "wednesday": (8.5, 19),
+            "thursday": (8.5, 19),
+            "friday": (8.5, 19),
+            "saturday": (0, 0),
+            "sunday": (0, 0),
+        }
+  
+  Valid hour values must be set between 0 and 23.9999.
+  In this example 8.5 is interpreted as 8:30AM, saturdays and sundays don't count.
+  
+- **HELPDESK_FOLLOWUP_TIME_SPENT_EXCLUDE_HOLIDAYS** List of days in format "%m-%d" to exclude from automatic follow-up 'time_spent' calculation.
+
+  **Default:** ``HELPDESK_FOLLOWUP_TIME_SPENT_EXCLUDE_HOLIDAYS = ()``
+  
+  This example removes the first and last days of the year together with Christmas::
+
+        HELPDESK_FOLLOWUP_TIME_SPENT_EXCLUDE_HOLIDAYS = ("01-01",
+                                                         "12-25",
+                                                         "12-31",)
```

Comment:
How would you implement it?
A callable that receives the year or the full FollowUp instance?

This example would become:
*lambda fup: ("01-01",  "12-25", "12-31",)*

Reply:
That may be more complicated than I though, the best would be to give the `earliest` and `latest` parameters since they could be on two different years. But then the function would need to return two different lists.

i wonder if it's worth it :sweat_smile:

Reply:
Or no, the callable only needs a year as the parameter and return the dynamic list of holidays for that year.

I think that would work

Reply:
Or we could include the year.

`("2024-01-01", "2024-12-25", "2024-12-31",)`

Reply:
Done with the year solution.

Reply:
Yes, good idea!  
And it's still possible for the user to have something to calculate the list of holidays for the next 20 years for example.

They'll use the setting like this : 
HELPDESK_FOLLOWUP_TIME_SPENT_EXCLUDE_HOLIDAYS = get_list_of_next_holidays()